docs/source/contribution.rst

```diff
@@ -2,29 +2,59 @@
 Contribution
 ++++++++++++
 
-People
-======
-
-+------------------+---------------------+
-| Name             | Role                |
-+==================+=====================+
-| KunYu Chen       | ``Project Founder`` |
-+------------------+---------------------+
-| JunWei Song      | ``Lead Developer``  |
-+------------------+---------------------+
-| AnWei Kung       | ``Core Developer``  |
-+------------------+---------------------+
-| Sin              | ``Malware Analyst`` |
-+------------------+---------------------+
-| IokJin Sih       | ``Malware Analyst`` |
-+------------------+---------------------+
-| Chun-I Fan       | ``Consultant``      |
-+------------------+---------------------+
-| HY Lin           | ``Consultant``      |
-+------------------+---------------------+
-| Jheng-Jia Huang  | ``Consultant``      |
-+------------------+---------------------+
-| Pippen Wang      | ``Consultant``      |
-+------------------+---------------------+
-| ChaoWen Li       | ``Consultant``      |
-+------------------+---------------------+
+Different stages of core members
+================================
+
+* ``Contributor``:
+  As soon as you post a comment, fix a typo, ask a question or report a bug etc.
+  , you become an active contributor.
+
+
+
+* ``Contributor followed by a mentor``:
+  Spotted active contributors can be asked to get a mentor to speedup their
+  learning of this project.
+
+
+* ``Core member``:
+  Once other core members consider that a contributor is ready to be promoted,
+  a core member opens a private vote for the candidate.
```

Comment:
How about this:

Once other core members consider that a contributor is ready to be promoted, a private vote between core members will be held for the candidate.